README.md

```diff
@@ -13,9 +13,11 @@ DataJoint LabBook is a graphical user interface built using React to facilitate
 
 If you have not done so already, please install the following dependencies:
 
+
```

Comment:
@ChihweiLHBird There is extra whitespace here and after the merge conflicts it looks like the docs section on **Run Tests w/ Docker** was removed.

Reply:
Merged, thanks!